python/arcticdb/util/storage_test.py

```diff
@@ -0,0 +1,241 @@
+from arcticdb import Arctic
+import pandas as pd
+import os
+import numpy as np
+import argparse
+import re
+from datetime import datetime
+
+
+def normalize_lib_name(lib_name):
+    lib_name = lib_name.replace(".", "_")
+    lib_name = lib_name.replace("-", "_")
+
+    return lib_name
+
+
+def real_s3_credentials(shared_path: bool = True):
+    endpoint = os.getenv("ARCTICDB_REAL_S3_ENDPOINT")
+    bucket = os.getenv("ARCTICDB_REAL_S3_BUCKET")
+    region = os.getenv("ARCTICDB_REAL_S3_REGION")
+    access_key = os.getenv("ARCTICDB_REAL_S3_ACCESS_KEY")
+    secret_key = os.getenv("ARCTICDB_REAL_S3_SECRET_KEY")
+    if shared_path:
+        path_prefix = os.getenv("ARCTICDB_PERSISTENT_STORAGE_SHARED_PATH_PREFIX")
+    else:
+        path_prefix = os.getenv("ARCTICDB_PERSISTENT_STORAGE_UNIQUE_PATH_PREFIX")
+
+    clear = True if str(os.getenv("ARCTICDB_REAL_S3_CLEAR")).lower() in ["true", "1"] else False
```

Comment:
No need for the ternary operator here.

```suggestion
    clear = str(os.getenv("ARCTICDB_REAL_S3_CLEAR")).lower() in ("true", "1")
```